### PR TITLE
Attach tokens to SDK client

### LINKED ISSUE #180: Attach tokens to SDK client
There are cases when session token or bearer token used with all requests. It may be a bit annoying to add token manually for each request. There is a proposal to attach tokens directly to client and reuse it with every request. 

Proposed interface:

```go
cli, err := client.New(key, client.WithAddress(nodeAddress))
token, err := cli.CreateSession(ctx, math.MaxUint64)

cli.AttachSessionToken(token) // AttachBearerToken should also be available

cli.GetObject(ctx, addr) // pre-attached session token is being used
cli.GetObject(ctx, addr, client.WithSessionToken(newToken) // new session token is being used
```

### PR DESCRIPTION
**Pull request by cthulhu-rider**
Closes #180.

### CODE REVIEW

**Comment by im-kulikov** [comment]
since we have `defaultOptions`, that includes session/bearer token, can we omit ʻowner.ID`?

for example 
```go
func (c Client) listContainerV2(ctx context.Context, owner *owner.ID, opts ...CallOption) ([]*container.ID, error) {
```

**Comment by alexvanin** [comment]
> since we have `defaultOptions`, that includes session/bearer token, can we omit ʻowner.ID`?
> 
> for example
> 
> ```go
> func (c Client) listContainerV2(ctx context.Context, owner *owner.ID, opts ...CallOption) ([]*container.ID, error) {
> ```
> 
> cc @alexvanin @cthulhu-rider

No, we can't. Both session and bearer tokens have nothing to do with containers. Client can ask information about containers from any owner since it is stored publicly in chain. To omit `owner.ID` use `client.ListSelfContainers()` function, that uses client key as owner source.

File: pkg/client/opts.go [hunk 1]
```diff
@@ -132,6 +134,8 @@ func v2MetaHeaderFromOpts(options callOptions) *v2session.RequestMetaHeader {
 		meta.SetBearerToken(options.bearer.ToV2())
 	}
 
+	meta.SetSessionToken(options.session.ToV2())
```
Comment (alexvanin) [inline pkg/client/opts.go:137]: Do we need that? Right now session token already processed in `attachV2SessionToken()` function.

**Review by alexvanin — APPROVED** [review]

**Comment by im-kulikov** [comment]
> > since we have `defaultOptions`, that includes session/bearer token, can we omit ʻowner.ID`?
> > for example
> > ```go
> > func (c Client) listContainerV2(ctx context.Context, owner *owner.ID, opts ...CallOption) ([]*container.ID, error) {
> > ```
> > 
> > 
> > cc @alexvanin @cthulhu-rider
> 
> No, we can't. Both session and bearer tokens have nothing to do with containers. Client can ask information about containers from any owner since it is stored publicly in chain. To omit `owner.ID` use `client.ListSelfContainers()` function, that uses client key as owner source.

mb it can be variadic? If options have `WithOwnerID()` then we use it, otherwise, we can take the owner from `bearer` or `session` tokens?

File: pkg/client/opts.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/client/opts.go:137]: Yes, `attachV2SessionToken` is called for object operations only.

**Pull request merged**